### ISSUE TITLE
Coverity job needs update

### ISSUE THREAD

**Issue by jbergstroem** [nodejs/build#780]
```
 Your request for analysis of Node.js  is failed.
   Analysis status: Failure
   Please fix the error and upload the build again.

Error details:
   The Coverity Build tool version is no longer supported.
   Please download the latest version for your platform <snip>
```

**Comment by nschonni** [comment]
May be related to this from the downloads page https://scan.coverity.com/download
```
WARNING: Linux users on kernel version 4.8.x and newer will need to apply a sysctl parameter to support our binaries. Without this parameter, our binaries will not work on your platform. This is a known issue which we will address with a future release.

# sysctl vsyscall=emulate
```

**Comment by jbergstroem** [comment]
I just updated the toolchain and are trying a rebuild with some minor adjustments. I've also added `deps/nghttp2` and `deps/node-inspect` to exclusion patterns.

Finally, how do we handle people that wants to see the result? As of now, it is invite only but there's two invites from april that haven't been approved 😞  -- I don't have any objections to accepting all invites seeing how its relatively trivial to reproduce the results. Opinions?
